src/models/Home.js

```diff
@@ -34,3 +34,7 @@ export type CategoryBalances = {|
 export type ChainBalances = ChainRecord<BigNumber>;
 
 export type CollectibleCountPerChain = ChainRecord<number>;
+
+export type AccountsTotalBalances = {
```

Comment:
Changed to `ChainBalancesPerAccount` maybe it would bring more sense as Balances confuses since we have Balances for Assets and Balances for Category (+ Category per Chain).